src/altimate.ts

```diff
@@ -170,6 +170,7 @@ interface DocsGenerateModelRequestV2 {
   };
   prompt_hint?: string;
   gen_model_description: boolean;
+  columns_count: number;
```

Comment:
variable naming could be improved. `current_column_index` or `completed_columns_count`?

Reply:
Agree

Reply:
I think along with this, we send an unique id. session_id.table_id.bulk_gen or somthing like that. 

https://github.com/AltimateAI/vscode-dbt-power-user/pull/1151/files#diff-f5d2312039a56f9d5d5c2eff4188b2ab44446ae157202ee46a51da2a5ef242c7R16

You can find it here

Reply:
You can store this in the backend and say (2000 executions) charged for (300) or smthg like that

Reply:
vscode_session_id-{model-unique-id}-{bulk.gen}